pkg/bmc/idrac_virtualmedia.go

```diff
@@ -90,3 +91,12 @@ func (a *redfishiDracVirtualMediaAccessDetails) VendorInterface() string {
 func (a *redfishiDracVirtualMediaAccessDetails) SupportsSecureBoot() bool {
 	return true
 }
+
+func (a *redfishiDracVirtualMediaAccessDetails) Validate(bmcCreds Credentials) error {
+	address := getRedfishAddress(a.bmcType, a.host)
+	vmediahttpClient := http.Client{
+		Transport: &http.Transport{},
+	}
+	return ValidateVirtualMedia(address, bmcCreds.Username, bmcCreds.Password,
```

Comment:
These are not enough, you need to pass the system ID.

Reply:
@dtantsur not sure if understand, we look for the system in the function

Reply:
Done